packages/profiling-browser/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@sentry/profiling-browser",
+  "version": "7.38.0",
+  "description": "Official Sentry SDK for browser profiling",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/profiling-browser",
+  "author": "Sentry",
+  "license": "MIT",
+  "engines": {
+    "node": ">=8"
+  },
+  "sideEffects": [
+    "./build/cjs/index.js",
+    "./build/esm/index.js"
+  ],
+  "main": "build/cjs/index.js",
+  "module": "build/esm/index.js",
+  "types": "build/types/index.d.ts",
+  "publishConfig": {
+    "access": "public"
```

Comment:
Just to confirm. Do we want to publish this?

Reply:
Yeah, we want to test this on our sentry dashboard - an alternative is we can also move this to a separate repo and just specify a gh dep which would also work for us